tests/util/network_protocol_data.py

```diff
@@ -60,13 +60,13 @@
 
 proof_of_space = ProofOfSpace(
     bytes32(bytes.fromhex("1fb331df88bc142e70c110e21620374118fb220ccc3ef621378197e850882ec9")),
-    G1Element(
+    G1Element.from_bytes(
```

Comment:
`blspy` does provide a constructor for `G1Element` and `G2Element` that takes bytes, I think it's a bit clearer to use the `from_bytes()` function